sdks/python/src/opik/cli/export_utils.py

```diff
@@ -0,0 +1,480 @@
+"""Common utilities for export functionality."""
+
+import csv
+import json
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional
+
+from rich.console import Console
+
+console = Console()
+
+# Constants for export command
+FALLBACK_PAGE_SIZE = 10  # Smaller page size for error recovery
+MAX_EXPERIMENT_ITEMS_PER_FETCH = 1000  # Maximum items to fetch per experiment
+
+
+def matches_name_pattern(name: str, pattern: Optional[str]) -> bool:
+    """Check if a name matches the given pattern using simple string matching."""
+    if pattern is None:
+        return True
+    # Simple string matching - check if pattern is contained in name (case-insensitive)
+    return pattern.lower() in name.lower()
+
+
+def is_experiment_id(pattern: str) -> bool:
```

Comment:
As mentioned in previous review round, let's move it to opik.id_helpers as `is_uuid7`.